.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Is this necessary?

Reply:
This will update the codecoverage or any other action if necessary

Reply:
Okay, also set this interval to `weekly` .